src/Functions/randDistribution.cpp

```diff
@@ -320,168 +320,294 @@ class FunctionRandomDistribution : public IFunction
-    FunctionDocumentation{
-    .description=R"(
-Returns a random number from the t-distribution.
-Accepts one parameter - degree of freedom.
-
-Typical usage:
-[example:typical]
-)",
-    .examples{
-        {"typical", "SELECT randStudentT(5) FROM numbers(100000);", ""}},
-    .category = FunctionDocumentation::Category::RandomNumber
-    });
-
-
-    factory.registerFunction<FunctionRandomDistribution<FisherFDistribution>>(
-    FunctionDocumentation{
-    .description=R"(
-Returns a random number from the f-distribution.
-The F-distribution is the distribution of X = (S1 / d1) / (S2 / d2) where d1 and d2 are degrees of freedom.
-Accepts two parameters - degrees of freedom.
-
-Typical usage:
-[example:typical]
-)",
-    .examples{
-        {"typical", "SELECT randFisherF(5) FROM numbers(100000);", ""}},
-    .category = FunctionDocumentation::Category::RandomNumber
-    });
-
-
-    factory.registerFunction<FunctionRandomDistribution<BernoulliDistribution>>(
-    FunctionDocumentation{
-    .description=R"(
-Returns a random number from the Bernoulli distribution.
-Accepts one parameter - probability of success.
-
-Typical usage:
-[example:typical]
-)",
-    .examples{
-        {"typical", "SELECT randBernoulli(0.1) FROM numbers(100000);", ""}},
-    .category = FunctionDocumentation::Category::RandomNumber
-    });
-
-
-    factory.registerFunction<FunctionRandomDistribution<BinomialDistribution>>(
-    FunctionDocumentation{
-    .description=R"(
-Returns a random number from the binomial distribution.
-Accepts two parameters - number of experiments and probability of success in each experiment.
-
-Typical usage:
-[example:typical]
-)",
-    .examples{
-        {"typical", "SELECT randBinomial(10, 0.1) FROM numbers(100000);", ""}},
-    .category = FunctionDocumentation::Category::RandomNumber
-    });
-
-
-    factory.registerFunction<FunctionRandomDistribution<NegativeBinomialDistribution>>(
-    FunctionDocumentation{
-    .description=R"(
-Returns a random number from the negative binomial distribution.
-Accepts two parameters - number of experiments and probability of success in each experiment.
-
-Typical usage:
-[example:typical]
-)",
-    .examples{
-        {"typical", "SELECT randNegativeBinomial(10, 0.1) FROM numbers(100000);", ""}},
-    .category = FunctionDocumentation::Category::RandomNumber
-    });
-
-
-    factory.registerFunction<FunctionRandomDistribution<PoissonDistribution>>(
-    FunctionDocumentation{
-    .description=R"(
-Returns a random number from the poisson distribution.
-Accepts one parameter - the mean number of occurrences.
-
-Typical usage:
-[example:typical]
-)",
-    .examples{
-        {"typical", "SELECT randPoisson(3) FROM numbers(100000);", ""}},
-    .category = FunctionDocumentation::Category::RandomNumber
-    });
+    FunctionDocumentation::Description description = R"(
+Returns a random number drawn uniformly from the interval $[\min, \max]$.
```

Comment:
Minor: `[...] a random Float64 number [...]`

(same comment applies in the other functions in the rest of the file)